wofi/README.md

```diff
@@ -0,0 +1,5 @@
+# Selenized color palette for Wofi
+
+A [Wofi](https://hg.sr.ht/~scoopta/wofi) color scheme. Wofi is a Wayland native replacement for
+
+Included is a basic config, the Wofi site has some [styling information](https://cloudninja.pw/docs/wofi.html). The [GTK+ CSS Supported properties](https://developer.gnome.org/gtk3/stable/chap-css-properties.html) is also helpful for knowing what to set.
```

Comment:
Can you please write where these files should be saved? This may be obvious to the Wofi users, but I think it's better to have it.

Reply:
Included, it was in the man file though https://hg.sr.ht/~scoopta/wofi/browse/default/man/wofi.1#L110

The man files are pretty good, the only thing this doesn't have is a sample working config (which is why I included one).

Reply:
Ah, ok. I've briefly looked at the link but missed it.